app/src/main/java/info/nightscout/android/medtronic/service/MedtronicCnlIntentService.java

```diff
@@ -244,8 +244,8 @@ protected void onHandleIntent(Intent intent) {
 
                         // Check if pump sent old event when new expected
                         if (dataStore.getLastPumpStatus() != null &&
-                                dataStore.getLastPumpStatus().getPumpDate() != null &&
-                                pumpRecord.getPumpDate().getTime() - dataStore.getLastPumpStatus().getPumpDate().getTime() < 5000L &&
+                                dataStore.getLastPumpStatus().getSgvDate() != null &&
+                                pumpRecord.getSgvDate().getTime() - dataStore.getLastPumpStatus().getSgvDate().getTime() < 5000L &&
```

Comment:
@Pogman 👍 then I don't need to open a PR for my commit which does the same thing :-)